src/core/lib/gprpp/arena.cc

```diff
@@ -31,11 +31,28 @@
 #include "src/core/lib/gpr/alloc.h"
 #include "src/core/lib/gprpp/memory.h"
 
+namespace {
+
 template <size_t alignment>
-static void* gpr_arena_malloc(size_t size) {
+void* gpr_arena_malloc(size_t size) {
```

Comment:
This function doesn't really add any value anymore.  Might as well remove it and have the callers invoke `gpr_malloc_aligned()` directly.

Reply:
Done.